ext/gd/gd.stub.php

```diff
@@ -520,6 +520,10 @@ function imagecolorexactalpha(GdImage $image, int $red, int $green, int $blue, i
 
 function imagecopyresampled(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_width, int $dst_height, int $src_width, int $src_height): bool {}
 
+#ifdef HAVE_GD_COPY_ROTATED
+function imagecopyrotated(GdImage $dst_image, GdImage $src_image, float $dst_x, float $dst_y, int $src_x, int $src_y, int $src_width, int $src_height, int $angle): bool {}
```

Comment:
angle should be float

Reply:
I follow gd api [here](https://github.com/libgd/libgd/blob/d11db230d3e8c48630e0abe24e34f0f12f45d989/src/gd.h#L1322).